app/src/main/java/com/example/formula1/view/mainscreen/fragments/standing/driverdetail/DriverStandingDetail.kt

```diff
@@ -62,9 +73,13 @@ class DriverStandingDetail :
     }
 
     private fun getData() {
-        val id = intent.getStringExtra(DRIVER_ID)
-        if (id != null) {
-            viewModel.searchDriver(id.toInt())
+        val driverID = intent.getStringExtra(DRIVER_ID)
```

Comment:
```suggestion
        val driverID = intent.getStringExtra(KEY_DRIVER_ID)
```
Thêm tiền tố kEY hoặc BUNDLE_ vào để dùng cho các kiểu data key-value nhé